src/main/java/org/broadinstitute/hellbender/tools/reference/ReferenceSequenceTable.java

```diff
@@ -0,0 +1,354 @@
+                int ref2Index = pair.getRef2ColumnIndex();
+                TableEntry[] entries = row.getEntries();
+
+                TableEntry ref1Value = entries[ref1Index];
+                TableEntry ref2Value = entries[ref2Index];
+                if (!ref1Value.getColumnValue().equals(ref2Value.getColumnValue())) {
+                    pair.removeStatus(ReferencePair.Status.EXACT_MATCH);
+                }
+                if(!ref1Value.getColumnValue().equals(ref2Value.getColumnValue()) &&
+                        !(ref1Value.getColumnValue().equals(MISSING_ENTRY) || ref2Value.getColumnValue().equals(MISSING_ENTRY))){
+                    pair.addStatus(ReferencePair.Status.DIFFER_IN_SEQUENCE_NAMES);
+                }
+            }
+        }
+        for(ReferencePair pair : refPairs){
+            boolean superset = false;
+            boolean subset = false;
+            for(String sequenceName : tableBySequenceName.keySet()){
+                int ref1Index = pair.getRef1ColumnIndex();
+                int ref2Index = pair.getRef2ColumnIndex();
+                Set<ReferenceSequenceTable.TableRow> rows = queryBySequenceName(sequenceName);
+                int sequenceNameFound = 0;
+
+                for(TableRow row : rows) {
+                    TableEntry[] entries = row.getEntries();
+                    TableEntry ref1Value = entries[ref1Index];
+                    TableEntry ref2Value = entries[ref2Index];
+
+                    if((ref1Value.getColumnValue().equals(sequenceName) ^ ref2Value.getColumnValue().equals(sequenceName))
+                            && (ref1Value.getColumnValue().equals(MISSING_ENTRY) ^ ref2Value.getColumnValue().equals(MISSING_ENTRY))){
+                        sequenceNameFound++;
+                    }
+                    if(ref1Value.getColumnValue().equals(MISSING_ENTRY) ^ ref2Value.getColumnValue().equals(MISSING_ENTRY)){
+                        if(ref1Value.getColumnValue().equals(MISSING_ENTRY)){
+                            subset = true;
+                        }
+                        else if(ref2Value.getColumnValue().equals(MISSING_ENTRY)){
+                            superset = true;
+                        }
+                    }
+                }
+
+                if(sequenceNameFound == 2){
+                    pair.addStatus(ReferencePair.Status.DIFFER_IN_SEQUENCE);
+                } else if(sequenceNameFound == 1){
+                    pair.addStatus(ReferencePair.Status.DIFFER_IN_SEQUENCES_PRESENT);
+                } else if (sequenceNameFound > 2){
+                    throw new UserException.BadInput(String.format("Duplicate of sequence '%s' found in %s or %s.", sequenceName, pair.getRef1(), pair.getRef2()));
+                }
+            }
+
+            if(superset ^ subset && !pair.getStatus().contains(ReferencePair.Status.DIFFER_IN_SEQUENCE_NAMES)){
+                pair.removeStatus(ReferencePair.Status.DIFFER_IN_SEQUENCES_PRESENT);
```

Comment:
Also add a comment here explaining the logic behind the superset/subset calculation